File: codechef/Num239.java

import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;

class Num239 {	
	public static void main(String[] args) throws NumberFormatException, IOException {
		BufferedReader In = new BufferedReader(new InputStreamReader(System.in));
	    int cases = Integer.parseInt(In.readLine());
	    for(int i=0; i<cases; i++) {
	        int count = 0;
	        String[] nums = In.readLine().split(" ");
	        for(int num=Integer.parseInt(nums[0]); num<=Integer.parseInt(nums[1]); num++) {
	            if(num % 10 == 2 || num%10 == 3 || num%10 == 9) {
	                count +=1;
	            }
	        }
	        System.out.println(count);
	    }
	}
}

File: codechef/Num239/num239.kt
package codechef

import java.io.BufferedReader
import java.io.InputStreamReader

fun main() {
    val In =  BufferedReader(InputStreamReader(System.`in`))
    val cases = In.readLine()!!.toInt()
    for(i in 1..cases) {
        var count = 0
        val nums = In.readLine()!!.split(" ").map { it.toInt() }
        for(num in nums[0]..nums[1]) {
            if(num % 10 == 2 || num%10 == 3 || num%10 == 9) {
                count++
            }
        }
        println(count)
    }
}